language/tools/vm-genesis/src/lib.rs

```diff
@@ -239,23 +235,44 @@ fn create_and_initialize_testnet_minting(
     );
 }
 
-/// Initialize each validator.
-fn create_and_initialize_validators(
+/// Creates and initializes each validator owner and validator operator.
+fn create_and_initialize_owners_operators(
     context: &mut GenesisContext,
-    validators: &[ValidatorRegistration],
+    owner_names: Vec<String>,
+    operator_registrations: &[OperatorRegistration],
 ) {
-    for (account_key, registration) in validators {
+    // Create accounts for each validator owner
+    for _owner_name in owner_names {
         context.set_sender(account_config::association_address());
 
-        let auth_key = AuthenticationKey::ed25519(&account_key);
-        let account = auth_key.derived_address();
-        let create_script = transaction_builder::encode_create_validator_account(
-            account,
-            auth_key.prefix().to_vec(),
+        // TODO(joshlind): use the owner_name to derive a unique and deterministic account address
+        // For now, we use a random auth key and account below:
+        let owner_auth_key = AuthenticationKey::random();
+        let owner_account = AccountAddress::random();
```

Comment:
how would you even know how to use this account if it is random? this commit probably won't pass smoke tests.

Reply:
This is fixed in subsequent commits. Trying to tackle one thing at a time 😄